website/docs/d/alert_configuration.html.markdown

```diff
@@ -121,8 +121,12 @@ In addition to all arguments above, the following attributes are exported:
 * `updated` - Timestamp in ISO 8601 date and time format in UTC when this alert configuration was last updated.
 * `enabled` - If set to true, the alert configuration is enabled. If enabled is not exported it is set to false.
 * `event_type` - The type of event that will trigger an alert.
+* `matcher` - Rules to apply when matching an object against this alert configuration. See [matchers](#matchers).
+* `metric_threshold_config` - The threshold that causes an alert to be triggered. Required if `event_type_name` : `OUTSIDE_METRIC_THRESHOLD` or `OUTSIDE_SERVERLESS_METRIC_THRESHOLD`. See [metric threshold config](#metric-threshold-config).
+* `threshold_config` - 	 Threshold that triggers an alert. Required if `event_type_name` is any value other than `OUTSIDE_METRIC_THRESHOLD` or `OUTSIDE_SERVERLESS_METRIC_THRESHOLD`. See [threshold config](#threshold-config).
+* `notifications` - List of notifications to send when an alert condition is detected. See [notifications](#notifications).
```

Comment:
Unifying docs of alert_configuration and alert_configurations docs: copied from plural data source